scripts/generate_errors.pl

```diff
@@ -48,7 +48,7 @@
                             CAMELLIA CCM CHACHA20 CHACHAPOLY CMAC CTR_DRBG DES
                             ENTROPY ERROR GCM HKDF HMAC_DRBG MD2 MD4 MD5
                             NET OID PADLOCK PBKDF2 PLATFORM POLY1305 RIPEMD160
-                            SHA1 SHA256 SHA512 THREADING XTEA );
+                            SHA1 SHA256 SHA512 SM3 THREADING XTEA );
 my @high_level_modules = qw( CIPHER DHM ECP MD
```

Comment:
Note: if this pull request is merged after https://github.com/ARMmbed/mbedtls/pull/4304 and before we have time to improve `tests/scripts/depends-hashes.pl`, SM3 will need to be added to that script.